assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -630,10 +630,20 @@ che.workspace.devfile.default_editor.plugins=eclipse/che-machine-exec-plugin/nig
 # Only secrets that match ALL given labels will be selected.
 che.workspace.provision.secret.labels=app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspace-secret
 
-
 # Plugin is added in case async storage feature will be enabled in workspace config
 # and supported by environment
 che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
 
 # Docker image for the Che async storage
 che.infra.kubernetes.async.storage.image=quay.io/eclipse/che-workspace-data-sync-storage:latest
+
+# Configures name of the trust-store config map where the self-signed certificates are stored.
+# This map is supposed to be created by Che installer (operator or etc), and it is supposed to
+# be located in the same namespace where the Che server deployed.
+che.trusted_ca_bundles_config_map=ca-certs
```

Comment:
since this is Openshift4 feature. why don't we rename properties to `che.infra.openshift.trusted.ca.bundles.*`?

Reply:
I think I agree with @skabashnyuk here, because this is currently an OpenShift4 specific feature. If we are going to implement it for the other platforms, it is most probably going to look differently and probably will be also configured in a different way?

Reply:
ok